packages/dataviews/src/dataform-controls/select.tsx

```diff
@@ -14,6 +14,8 @@ import getCustomValidity from './utils/get-custom-validity';
 
 const { ValidatedSelectControl } = unlock( privateApis );
 
+const QUERY_ALL_RESULTS = { perPage: -1 };
```

Comment:
I guess this is `DEFAULT_QUERY`, it's shared across the package, and we set `perPage` to 20 (or something along those lines)? The purpose of this stream of work is improving performance by loading only the set of elements we are using. If we fetch them all, we're doing the same as before.

Reply:
For Edit, yes. As I've already commented, I think the PR would be quite huge if I try to add a component for edit too. If you think this is integral part, I can try and see what design feedback we have and explore it.